manifests/metallb-frr.yaml

```diff
@@ -497,6 +497,8 @@ spec:
               value: /etc/frr_reloader/reloader.pid
             - name: METALLB_BGP_TYPE
               value: frr
+            - name: FRR_LOGGING_LEVEL
```

Comment:
We should document this somehow, even if it's a comment block in this manifest for now.  I'd expect it to list what the default value is, and what other valid choices are